be/src/common/config.h

```diff
@@ -769,4 +769,10 @@ CONF_mBool(dependency_librdkafka_debug_enable, "false");
 // admin, eos, mock, assigner, conf
 CONF_String(dependency_librdkafka_debug, "all");
 
+// max loop count when be waiting its fragments finish
+CONF_Int64(be_loop_count_wait_fragments_finish, "0");
```

Comment:
I think the BE and CN could use one config.

Reply:
changed to loop_count_wait_fragments_finish